src/data/data-web3.html

```diff
@@ -264,6 +267,43 @@
             }
 
             /**
+             * Returns an ipfsAddress given a HashtagAddress
+             * @param {string} contractName The contract where the lookup is queried
+             * @param {string} lookupHashtag the address of the hashtag to get the
+             * metadata ipfs address for
+             * @return {ipfsAddress} ipfsAddress is a string
+             */
+            _getHashTagIpfsTag(contractName, lookupHashtag) {
+                return new Promise((resolve, reject) => {
+                    Promise.all(
+                        [this._loadAbi(contractName),
+                            this._loadAddress(contractName),
+                            this._loadProviders()]
+                    ).then((data) => {
+                        // constructing contract from abi/address should happen globally
+                        const abiArray = data[0].abi;
+                        const contractAddress = data[1][contractName];
+                        const providers = data[2]['ropsten'];
```

Comment:
Is specifying ropsten here the best place of should it be an argument to this function?